artifact-definition-metaschema.json

```diff
@@ -63,6 +63,38 @@
             ]
           },
           "properties": {
+            "cloud": {
+              "type": "object",
+              "description": "Properties of the cloud supported by Massdriver. Only valid for 'credential' artifact definitions.",
+              "required": [
+                "regions"
+              ],
+              "properties": {
+                "id": {"type": "string", "title": "ID", "description": "Identifier for cloud"},
+                "regions": {
```

Comment:
Allow regions to be set in MD via the credential config for a cloud. Lets community and ops team add regions until we have cloud query up and running.